applications/consdb/values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for consdb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of web deployment pods to start
+replicaCount: 1
+
+image:
+  # -- Image to use in the consdb deployment
+  repository: "ghcr.io/lsst-dm/consdb-hinfo"
+
+  # -- Pull policy for the consdb image
+  pullPolicy: "IfNotPresent"
+
+  # -- Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+kafka:
+  # -- Kafka bootstrap server
+  KAFKA_BOOTSTRAP: "sasquatch-base-kafka-bootstrap.lsst.codes"
+  SCHEMA_URL: "https://sasquatch-base-kafka-schema-registry.lsst.codes"
+  # -- username for SASL_PLAIN authentication
+  KAFKA_USERNAME: "consdb"
+  # -- password for SASL_PLAIN authentication
+  KAFKA_PASSWORD: ""
```

Comment:
You don't want the password as part of the Helm values; it'll get routed through VaultSecrets/Secrets into the Deployment directly.

Reply:
ok I will remove - though I like this its explicit ..